lib/api/v1/namespaces.rb

```diff
@@ -11,11 +11,12 @@ class Namespaces < Grape::API
           authorization!(force_admin: false)
         end
 
-        helpers ::API::Helpers::Namespaces
-
         desc "Returns a list of namespaces",
              tags:     ["namespaces"],
-             detail:   "This will expose all accessible namespaces",
+             detail:   "This will expose all accessible namespaces by the user via either team
+                        membership or visibility. Keep in mind that if the user is an admin, this
+                        will return all the global, personal and other namespaces created by all
+                        the users.",
```

Comment:
Nitpick: remove the "." at the end. When exporting to the documentation we expect all strings to not have a period at the end.